cpp/src/groupby/new_groupby.cu

```diff
@@ -5,6 +5,9 @@
 #include "utilities/error_utils.h"
 #include "aggregation_operations.hpp"
 #include "groupby/hash_groupby.cuh"
+
+#include <thrust/fill.h>
```

Comment:
Is this include necessary? I don't see any `thrust::fill` calls in the changes to this file.